### ISSUE TITLE
Error in fastjet when processing masked array

### ISSUE THREAD

**Issue by lgray** [scikit-hep/fastjet#64]
Using the fastjet build within the container:
```
/cvmfs/unpacked.cern.ch/registry.hub.docker.com/coffeateam/coffea-dask:latest
```

`numpy` is `1.21.1`
`awkward` is `1.4.0`
`fastjet` is `3.3.4.0rc6`
`vector` is `0.8.4`

The following code:
```python3
import fastjet
import vector
import awkward as ak
import numpy as np
import uproot

vector.register_awkward()

nano = uproot.open("http://t3serv001.mit.edu/~freerc/002B7427-73F0-6741-8A45-80FA4886B115.root")

events = nano["Events"].arrays(["PFCands_trkPt", "PFCands_trkEta", "PFCands_trkPhi", "PFCands_mass", "PFCands_fromPV"])

Cands = ak.zip({
      "pt": events.PFCands_trkPt,
      "eta": events.PFCands_trkEta,
      "phi": events.PFCands_trkPhi,
      "mass": events.PFCands_mass
    }, with_name="Momentum4D")

cleaned_cands = Cands[(events.PFCands_fromPV > 1) & (events.PFCands_trkPt >= 1) & (np.abs(events.PFCands_trkEta) <= 2.5)]

#cleaned_cands = ak.packed(cleaned_cands)

jetdef = fastjet.JetDefinition(fastjet.antikt_algorithm, 1.5)
cluster = fastjet.ClusterSequence(cleaned_cands, jetdef)

jets = cluster.inclusive_jets(min_pt=3.0)
```

results in:

```
Traceback (most recent call last):
  File "total_chad.py", line 25, in <module>
    cluster = fastjet.ClusterSequence(cleaned_cands, jetdef)
  File "/opt/conda/lib/python3.8/site-packages/fastjet/__init__.py", line 218, in __init__
    fastjet._pyjet.AwkwardClusterSequence.__init__(
  File "/opt/conda/lib/python3.8/site-packages/fastjet/_pyjet.py", line 29, in __init__
    self._internalrep = fastjet._generalevent._classgeneralevent(data, jetdef)
  File "/opt/conda/lib/python3.8/site-packages/fastjet/_generalevent.py", line 11, in __init__
    self.multi_layered_listoffset(self.data)
  File "/opt/conda/lib/python3.8/site-packages/fastjet/_generalevent.py", line 78, in multi_layered_listoffset
    self.multi_layered_listoffset(ak.Array(data.layout.content))
  File "/opt/conda/lib/python3.8/site-packages/fastjet/_generalevent.py", line 78, in multi_layered_listoffset
    self.multi_layered_listoffset(ak.Array(data.layout.content))
  File "/opt/conda/lib/python3.8/site-packages/fastjet/_generalevent.py", line 63, in multi_layered_listoffset
    elif self._check_listoffset_subtree(ak.Array(data.layout.content)):
AttributeError: 'awkward._ext.RecordArray' object has no attribute 'content'
```

However if you uncomment the line:
```python3
#cleaned_cands = ak.packed(cleaned_cands)
```

The code functions perfectly. This appears to be a bug in array type handling within fastjet?

@nsmith- @chadfreer @jpivarski @aryan26roy

**Comment by jpivarski** [comment]
It's `contents` with an "s." This must have snuck through an earlier pass.

**Comment by aryan26roy** [comment]
@lgray I have been trying to reproduce the error but the script keeps getting stuck. Can you post a screenshot of the layout of the final awkward array?

**Comment by aryan26roy** [comment]
```python
<ListOffsetArray64>
    <offsets><Index64 i="[0 81 145 211 257 289 350 405 475 598 ... 2924456 2924525 2924576 2924636 2924700 2924766 2924855 2924907 2924941 2925015]" offset="0" length="42212" at="0x0000042fa1c0"/></offsets>
    <content><IndexedArray64>
        <index><Index64 i="[362 367 368 369 370 371 372 373 374 375 ... 88992599 88992600 88992601 88992602 88992603 88992604 88992618 88992708 88992738 88992771]" offset="0" length="2925015" at="0x000004eefd10"/></index>
        <content><RecordArray length="88994579">
            <parameters>
                <param key="__record__">"Momentum4D"</param>
            </parameters>
            <field index="0" key="pt">
                <NumpyArray format="f" shape="88994579" data="0.558594 0.567871 0.829102 0.599609 0.791504 ... 0 0 0 0 0" at="0x7fccb23d7010"/>
            </field>
            <field index="1" key="eta">
                <NumpyArray format="f" shape="88994579" data="1.19434 1.45801 -2.18359 1.20508 -0.0915527 ... -99 -99 -99 -99 -99" at="0x7fcc9d05a010"/>
            </field>
            <field index="2" key="phi">
                <NumpyArray format="f" shape="88994579" data="-2.32227 -2.6582 0.153687 1.07617 -2.70312 ... -99 -99 -99 -99 -99" at="0x7fcd1ac83010"/>
            </field>
            <field index="3" key="mass">
                <NumpyArray format="f" shape="88994579" data="0.139526 0.139526 0.139526 0.139526 0.139526 ... 0 0 -1.90735e-06 -9.53674e-07 0" at="0x7fcd05906010"/>
            </field>
        </RecordArray></content>
    </IndexedArray64></content>
</ListOffsetArray64>
```

**Comment by aryan26roy** [comment]
Update: was able to test it on your data @lgray . It works!

**Comment by aryan26roy** [comment]
FIxed in #63

**Comment by aryan26roy** [comment]
Fixed in #63 .
